e2e-test/src/tests/transform.test.js

```diff
@@ -0,0 +1,49 @@
+import { vBtn, vListTile } from 'jest-puppeteer-vuetify';
+import { uploadDataset } from '../util';
+
+
+describe('download dataset', () => {
```

Comment:
Can you change the describe block for this and the ANOVA test? They both still say `download dataset`